documentation/docs/get-started/llm-studio-flow.md

```diff
@@ -1,3 +1,6 @@
+---
+description: The flow of creating and fine-tuning large language models using H2O LLM Studio can be summarized in five sequential steps.
```

Comment:
Since we are not explaining the five steps within the description, I think we can change this one to something else.

Reply:
Fixed. All our other docs have a similar description. Could I change all of them to reflect the new change?